packages/common/src/ui/components/domain/StatusCrumbs/StatusCrumbs.tsx

```diff
@@ -42,6 +50,30 @@ export const StatusCrumbs = <StatusType extends string>(
     return acc;
   }, 0);
 
+  let Crumbs = null;
+  if (isSmallScreen) {
+    const stepKey = statuses[currentStep];
+    if (!stepKey) return null;
+    Crumbs = (
+      <Box flexDirection="row" display="flex" gap={1} justifyContent="center">
+        <StyledText>{t('label.status')}</StyledText>
+        <StyledText color={'secondary'}>
+          {t(statusFormatter(stepKey))}
+        </StyledText>
+      </Box>
+    );
+  } else {
+    Crumbs = statuses.map(status => {
+      const date = statusLog[status];
+
+      return (
+        <StyledText color={date ? 'secondary' : 'gray.main'} key={status}>
+          {t(statusFormatter(status))}
+        </StyledText>
+      );
+    });
+  }
```

Comment:
can see that this looks a bit ugly. Though, I think it's OK as a solution 🤷

Reply:
looks fine to me 🤷